### PR TITLE
refactor: rename `get_access_token` function to `get_bearer_token`

### PR DESCRIPTION
**Pull request by BryanttV**
### Description
This PRs renames `get_access_token` function to `get_bearer_token` and use `HTTPStatus` library to validate HTTP status codes

### CODE REVIEW

**Review by felipemontoya — CHANGES_REQUESTED** [review]
The idea is not to "rename" is to add both methods. Also test this because probably the code is broken as it stands right now.

File: sample-skill/lambda/lambda_function.py [hunk 1]
```diff
@@ -78,7 +79,7 @@ def get_access_token() -> tuple:
 
 def get_course_progress(username: str, coursename: str) -> str:
     """Return the progress of an user in a course"""
-    access_token, error_message = get_access_token()
+    access_token, error_message = get_jwt_token()
```
Comment (mariajgrimaldi) [inline sample-skill/lambda/lambda_function.py]: are you getting the JWT access token? Or a bearer token? where do we specify the JWT type?

Comment (BryanttV) [reply sample-skill/lambda/lambda_function.py]: Currently, the Bearer token is obtained, I will rename the function.

**Comment by BryanttV** [comment]
@mariajgrimaldi @felipemontoya, the eox-core and enrollments API accept Bearer token, therefore, I changed the function name to `get_bearer_token()`, and in this case it would not be necessary to add a new function to obtain the JWT token.

**Review by mariajgrimaldi — APPROVED** [review]

**Comment by felipemontoya** [comment]
The point I was trying to make is that eox-core APIs will require a Bearer token. CourseEnrollment APIs in turn will not work with Bearer and require a JWT token. For this reason we need methods to create BOTH types of token.

I think we should merge this as is and create a new PR that add the new funtion.

**Pull request merged**